src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -100,13 +112,17 @@ def consume_scenes(scene_list: list) -> list:
 
         scene_data.append(current_scene_data)
 
-    validate_scene_ids(scene_list)
-    validate_start_times(scene_list)
+    errors.extend(validate_scene_ids(scene_list))
+    errors.extend(validate_start_times(scene_list))
 
-    return scene_data
 
+    if errors:
+        errors.insert(0, f'Errors encountered while consuming scene with ID: {scene["scene_id"]}')
```

Comment:
This will only signal errors occurred in the last scene to be processed as it's outside of our loop of scenes. I think we need to do something clever here in order to get all the errors for each scene that has errors.

Reply:
I thought I tested it and it worked, but I'm not 100% sure, I'll make sure to fix it so this works. thanks for pointing it out

Reply:
Fixed it so it will insert for every scene that has errors.